src/App.js

```diff
@@ -116,6 +131,26 @@ const App = () => {
                         <Route path="/createBook" element={<CreateBook />} />
                     </Routes>
                 </InputContext.Provider>
+                <div>
+                    <div className="buttons">
+                        <button
+                            className="btn get-quote"
+                            onClick={getRandomQuote}
+                        >
+                            Loading Spinner Quote Button (click here)
+                        </button>
+                    </div>
+                    {loading ? (
+                        <LoadingSpinner />
+                    ) : (
```

Comment:
This behaviour should probably be hidden inside` LoadingSpinner `component with `loading` and `setLoading` passed as prop if I understood ticket correctly.

Reply:
Remove ability to click on page while loading.